<launch>
  <group ns="prosilica">
    <include file="cam_settings.xml"/>
    <param name="acquisition_mode" type="str" value="Continuous"/>
    <param name="ip_address" type="str" value="10.68.7.20"/>
  </group>
  <node name="prosilica" pkg="prosilica_cam" type="prosilica_node" respawn="false" output="screen"/>
</launch>
